src/components/PostList.js

```diff
@@ -56,6 +57,9 @@ class PostList extends React.Component {
 
     return <ul className='posts'>
       {pending && <li className='loading'>Loading...</li>}
+      {newPosts && <li>
+        <div className='post new-posts'>New Posts are available. <a onClick={newPosts}>Refresh</a></div>
```

Comment:
you'll definitely want to animate this (see my use of `velocity-react` elsewhere in the code)